src/MeetingsSDKAdapter.js

```diff
@@ -181,7 +183,15 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         this.meetings[ID] = {...meeting, localShare: stream};
         break;
       case MEDIA_TYPE_REMOTE_SHARE:
-        this.meetings[ID] = {...meeting, remoteShare: stream};
+        this.meetings[ID] = {...meeting, remoteShareStream: stream};
```

Comment:
Since this is the only event we get the with remote share stream, I'm storing it in a property on the meeting object and then using it when we get the stop/start events.